src/wp-includes/post.php

```diff
@@ -62,7 +62,16 @@ function create_initial_post_types() {
 			'rewrite'               => false,
 			'query_var'             => false,
 			'delete_with_user'      => true,
-			'supports'              => array( 'title', 'editor', 'author', 'thumbnail', 'page-attributes', 'custom-fields', 'comments', 'revisions' ),
+			'supports'              => array(
+				'title',
+				'author',
+				'thumbnail',
+				'page-attributes',
+				'custom-fields',
+				'comments',
+				'revisions',
+				'editor' => array( 'default_mode' => 'template-locked' ),
```

Comment:
```suggestion
				'editor' => array( 'default-mode' => 'template-locked' ),
```

Do we need to update to a new key?

- https://github.com/WordPress/gutenberg/pull/68745

Reply:
Yes and I also need to sync more recent changes ad well. I’ll do that today.